webpack/assets/javascripts/react_app/components/AuditsList/index.js

```diff
@@ -52,11 +52,13 @@ const renderResourceLink = (auditTitle, auditTitleUrl, id) => {
   return auditTitle;
 };
 
-const AuditsList = ({ data: { audits } }) => (
-  <ListView>
-    {audits.map(
-      (
-        {
+const AuditsList = ({ data: { audits } }) => {
+  const initExpanded = audits.length === 1;
```

Comment:
noticed that `initExpanded` doesn't  dynamic, so if audits prop will be changed (i.e via a future async action), `initExpanded` won't be updated. ATM it works fine because the audit list is given by server rendering.

Reply:
Why isn't dynamic?
It is based on props, what do I miss?

Reply:
the `initExpanded` value is calculated once the component is mount, so if `audits` would be changed through a future async action, the initExpanded is already calculated. transforming it into a function 
```js
const initExpanded = () => audits.length === 1;
```
will re-calculate that data.

Reply:
Mmmm... I think you confuse here, it will create a new const each time it rerenders.

But anyway, in this spesific case, I think patternfly will use the `initExpanded` for the initial-value of their inner state so once you update the `initExpanded` it won't affect anyway.

Reply:
oops, indeed, there is no a `render` method here, it's a functional component :)